systems/data-plan/api-gateway/pkg/rest/router.go

```diff
@@ -94,14 +93,14 @@ func (r *Router) init() {
 
 	baseRates := v1.Group("/baseRates", "BaseRates", "BaseRates operations")
 	baseRates.POST("", formatDoc("Upload baseRates", ""), tonic.Handler(r.uploadBaseRateHandler, http.StatusCreated))
-	baseRates.GET("/:baseRate", formatDoc("Get BaseRate", ""), tonic.Handler(r.getBaseRateHandler, http.StatusOK))
+	baseRates.GET("/:base_rate", formatDoc("Get BaseRate", ""), tonic.Handler(r.getBaseRateHandler, http.StatusOK))
 	baseRates.GET("", formatDoc("Get BaseRates by country", ""), tonic.Handler(r.getBaseRatesHandler, http.StatusOK))
 
 	packages := v1.Group("/packages", "Packages", "Packages operations")
 	packages.POST("", formatDoc("Add Package", ""), tonic.Handler(r.AddPackageHandler, http.StatusCreated))
 	packages.GET("/", formatDoc("Get packages", ""), tonic.Handler(r.getPackagesHandler, http.StatusOK))
 	packages.GET("/:package", formatDoc("Get package", ""), tonic.Handler(r.getPackageHandler, http.StatusOK))
-	packages.PATCH("/:package", formatDoc("Update Package", ""), tonic.Handler(r.UpdatePackageHandler, http.StatusOK))
+	packages.PATCH("/", formatDoc("Update Package", ""), tonic.Handler(r.UpdatePackageHandler, http.StatusOK))
```

Comment:
Why aren't we passing the ID of the package to update as path param anymore?  e.g. why aren't we doing `packages.PATCH("/:package",` anymore?

Reply:
Package `uuid` is include in payload body

`json
{
  "active": true,
  "data_volume": 0,
  "duration": 0,
  "name": "string",
  "org_rates_id": 0,
  "sim_type": "string",
  "sms_volume": 0,
  "uuid": "string",
  "voice_volume": 0
}
`

Reply:
This will work obviously. But then that endpoint is no more  REST, because we are mutating something without providing a URI.